packages/components/src/components/scrollable/scrollable.ts

```diff
@@ -0,0 +1,316 @@
+import { css, html, unsafeCSS } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { property, state } from 'lit/decorators.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import InteractiveStyles from '../../styles/interactive/interactive.css?inline';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Scrollable is used to indicate there is hidden content to be scrolled.
+ * @documentation https://solid.union-investment.com/[storybook-link]/scrollable
+ * @status stable
+ * @since 1.0
+ *
+ * @slot - The scrollable's content.
+ * @slot icon-start - The scrollable's start icon.
+ * @slot icon-end - The scrollable's end icon.
+ *
+ * @event button-left - Emitted when the left button is clicked.
+ * @event button-right - Emitted when the right button is clicked.
+ * @event button-up - Emitted when the top button is clicked.
+ * @event button-down - Emitted when the bottom button is clicked.
+ * @event end - Emitted when the end of the scrollable is reached.
+ * @event start - Emitted when the start of the scrollable is reached.
+ *
+ * @csspart base - The scrollable's base wrapper.
+ * @csspart button-start - The scrollable's start scroll button.
+ * @csspart button-end - The scrollable's end scroll button.
+ * @csspart shadow-left - The scrollable's left shadow.
+ * @csspart shadow-right - The scrollable's right shadow.
+ * @csspart shadow-top - The scrollable's top shadow.
+ * @csspart shadow-bottom - The scrollable's bottom shadow.
+ *
+ * @cssproperty --gradient-color - Defines a custom color for the gradient.
+ */
+
+@customElement('sd-scrollable')
+export default class SdScrollable extends SolidElement {
+  /** Defines the scroll orientation */
+  @property({ type: String, reflect: true }) orientation: 'horizontal' | 'vertical' | 'auto' = 'horizontal';
+
+  /** Activates scroll buttons */
+  @property({ type: Boolean, reflect: true }) buttons = false;
```

Comment:
I find it irritating, that the gradient is bound to the button. Is this really on purpose and should there be no way, that the gradient can be used without the button?

@coraliefeil ?

Reply:
If design is fine that, that there will never be gradients without buttons – let's go for it. :)